File: mp3player.cpp
#include "pxt.h"

namespace MP3-Modul {
	
    //%
	void sendCommand(Buffer buffer){
		//static const uint8_t cmd[] = {0x7E, 0xFF, 0x06, command, 0x00, hbyte, lbyte, 0xEF };
		//uBit.serial.send(cmd, 8);
		if (!buffer) return;
		
		ManagedBuffer buf(buffer);
      		uBit.serial.send(buf.getBytes(), buf.length());
	}

}

File: mp3player.ts

//% color=#0fbc11 icon="\uf120" block="mp3player"
namespace mp3player {

    /**
     * @param command buffer to send
     */
    //% block="serial|send|command %command" 
    //% blockId=send_command
    //% shim=mp3player::sendCommand
    export function send(command: Buffer): void {
        //mp3player.sendCommand(command, hbyte, lbyte)
        return;
    }
}
